docs/source/ml.rst

```diff
@@ -77,6 +77,21 @@ method. If the class you're using doesn't have a ``predict_proba`` method,
 you can add one by wrapping it in a :class:`~sklearn.calibration.CalibratedClassifierCV`.
 
 
+Custom Dependencies
+-------------------
```

Comment:
Perhaps this should link to the pip documentation somewhere, e.g. https://pip.pypa.io/en/stable/reference/pip_install/#vcs-support . This section of the documentation should probably mention that what we're running is `pip install [your input here]`.